README.md

```diff
@@ -1,3 +1,7 @@
+<a href="https://terraform.io">
+    <img src=".github/tf.png" alt="Terraform logo" title="Terraform" align="left" height="50" />
```

Comment:
I'm not sure that this path will work. May need to place in github.com/hashicorp/.github repo and then get the url.

Reply:
<img width="806" alt="image" src="https://user-images.githubusercontent.com/7771363/149206064-6e8e771e-faa4-45ff-be32-9d1fca6a4a8d.png">

Reply:
```html
<a href="https://terraform.io">
    <img src=".github/terraform_logo.svg" alt="Terraform logo" title="Terraform" align="right" height="50" />
</a>
```

Add `terraform_logo.svg` file from https://github.com/hashicorp/terraform-provider-aws/blob/main/.github/terraform_logo.svg